third-party/CMakeLists.txt

```diff
@@ -21,22 +21,32 @@ message("Entering ${CMAKE_CURRENT_SOURCE_DIR}")
 
 option(USE_SYSTEM_CLI11 "Use the CLI11 lib of the operating system" OFF)
 if(USE_SYSTEM_CLI11)
-  find_package(CLI11 "2.2.0" CONFIG REQUIRED)
-  set_target_properties(CLI11::CLI11 PROPERTIES IMPORTED_GLOBAL TRUE)
+  find_package(CLI11 "2.4.2" CONFIG REQUIRED)
+#  set_target_properties(CLI11::CLI11 PROPERTIES IMPORTED_GLOBAL TRUE)
```

Comment:
Why are we commenting this out ? If its not necessary, just remove it (but mention in the pr why it is now unnecessary)

Reply:
I was not sure it will work, now they have been all removed.

Reply:
I don't know if we want to keep, yes or no the third-party folder.
and what about licenses mention.

Reply:
Third-party license can be retrieve with `make write-licenses`